### DISCUSSION TITLE
PC not recognising cart reader

### DISCUSSION THREAD

**Discussion by v-flashpoint** [sanni/cartreader#763] [Hardware]
Hi all,

I'm running firmware 9.1 at the moment, and wanted to update to the latest, but oddly, when I plug my arduino into my PC, it's not being recognised at all. I have the driver installed, and I've flashed it a few times before, so this is a new one to me. The only changes I've made recently were attaching the N64 controller port, and finally fitting the standoffs. Is it possible something could have gone wrong with either of those?

Thanks!

Ian

**Comment by sanni** [comment]
I had this once. I fixed it by manually uninstalling the driver from the device manager and letting Windows 11 find the driver on its own via Windows Update. I'm assuming the lights on the Mega Pro still light up?

**Reply by v-flashpoint** [reply]
Thanks for the reply! Yeah, it all lights up fine. It just ends up either booting to the main menu, or giving me an SD error if there's not one in. No "something's been connected" noise from Windows either. Afraid I don't seem to be able to uninstall from device manager either, as it's not showing up there. Wonder what I've done?

**Reply by v-flashpoint** [reply]
I'm guessing it might be some weird driver issue on my end. Clicking "show hidden devices" doesn't show the right driver, and trying to uninstall the driver using the package doesn't work either (says "no device found"). Yet when you try to install the driver, it says it's already installed. Weird!

**Reply by sanni** [reply]
Maybe try uninstalling it while the Cart Reader is not connected with show hidden devices and then when uninstalling tick the try to remove driver from system box too.

I also found this in a txt file sitting on my PC, but I'm not sure if it was related to this or to something else:
````
1. Type "cmd" in Start Menu, right-click “Command Prompt” and choose “Run as Administrator”
2. Enter "set devmgr_show_nonpresent_devices=1" 
3. Enter "start devmgmt.msc"
4. In the box that opens, select "Show hidden devices" in the "view" menu.
Now if you expand the section on COM ports, all the COM ports that have ever
been created will be displayed, the non present ones being in grey. You can
uninstall away anything that you don’t want (right click, select uninstall).
````

**Reply by v-flashpoint** [reply]
Thanks again for the reply. Looks like it was pebkac though - just a crappy USB lead. I'd actually tried two so had assumed it couldn't be the issue, but just tried a third and it works fine. Thanks again though!
